### ISSUE TITLE
Bug in sr3 sanity - not finding missing instances

### ISSUE THREAD

**Issue by cleaaum** [MetPX/sarracenia#541]
sr3 sanity does no longer find the missing instances it should find. This is something I changed a few weeks ago because it was causing a crash with certain cases but now it is not working properly. Just changing it back. Also realizing that that sr3 sanity is always trying to start the post configs even though they are stopped and it probably shouldnt be? seems like another bug - tried to fix this already with this commit 2 weeks ago. [96b9919](https://github.com/MetPX/sarracenia/commit/96b991928b47797c23d22191b5de523b96d574d3)

**Comment by cleaaum** [comment]
Yeah I wanted to do a PR right away but was wondering if I should fix that 2nd issue at the same time.

**Comment by petersilva** [comment]
@reidsunderland I think given the recent stuff you were looking at, it might be good for you to take this one over.
I'm not sure if it isn't fixed already...

**Comment by petersilva** [comment]
I don't think this issue is relevant to current code base any more.
